### DISCUSSION TITLE
Dockerize and publish to docker hub

### DISCUSSION THREAD

**Discussion by AbeKline** [jas3333/GPT-Helper#21] [Ideas]
Is there any chance you could dockerize this and publish to docker hub? I'd love to run it on my Unraid server

**Comment by jas3333** [comment]
Yeah I'm planning on doing that once I learn docker. I'm working on another personal project that will involve docker so eventually I'll get to it.

**Comment by AbeKline** [comment]
Rad! If I get one put together would you want me to give you what I ended up with?

**Reply by jas3333** [reply]
I know this is a really late reply sorry about that. Been going in circles it seems trying to learn all this stuff. I finally put together a container for the updated GPT-Helper. It uses Pinecone and Mongodb for an insane memory. 

[Docker repo](https://hub.docker.com/repository/docker/jas313/gpt-chatbot/general)

**Reply by AbeKline** [reply]
Thank you! 
And yeah, early pandemic I went through this same journey figuring out docker/hub and holy crap is it annoying.
